src/components/RenderList.vue

```diff
@@ -9,17 +9,17 @@
           <strong>{{ingredient.name}}</strong>
           <span>Weight:</span>  
           <input placeholder="Edit" v-model.number="ingredient.quantity" type ="number" />
-          <span>Price:</span>
+          <span>Price/Kg:</span>
           <input placeholder="Edit" v-model.number="ingredient.price" type ="number" />
           <span>VAT:</span>
           <select v-model.number="ingredient.vat" >
```

Comment:
Refactor HTML to use `div` instead of `ul` and `li`, 
Use `label` element for input labels

Reply:
Use headings

Reply:
✔️